### ISSUE TITLE
Installation of cvxpy on pycharm is successful but fails on run time

### ISSUE THREAD

**Issue by isaeed3** [cvxgrp/cvxpy#695]
**Describe the bug**
cvxpy not working on pycharm

**To Reproduce**
Go to the pycharm terminal, install cvxpy and nose and run nosetests
gives error ImportError: No module named '_cvxcore'

**Version**
 - Python 3.6
 - OS: Windows
 - CVXPY Version: 1.0.21

**Comment by SteveDiamond** [comment]
We don't maintain the pycharm version of cvxpy, nor do we suggest installing via pycharm on Windows. You should take it up with pycharm.

**Comment by rileyjmurray** [comment]
Just following up on this. As Steven said, we do not provide support for IDE's like pycharm. However I can provide some guidance, since I personally use pycharm for cvxpy development.

My advice is this: handle package installation from the command-line (i.e. via conda install or pip install), and point your pycharm project interpreter to the appropriate virtual environment (pycharm supports environments managed by virtualenv, and anaconda). Once you install from the command line, test the installation by running python from the command line, and executing the single statement ``import cvxpy''. If that works, then a correctly configured pycharm development environment will also work.

**Comment by isaeed3** [comment]
I tried all possible methods to get cvxpy to run on pycharm but it didn't. All other packages work so it might have something to do with cvxpy that's why i posted here
I installed cvxpy on anaconda environment using pip (cvxpy worked on jupyter notebook, on pycharm, no issues with *import cvxpy* but always give this import error _cvxcore on runtime).
Also tried installing with python36 with pip install cvxpy, successfully installed package but gets the same import error _cvxcore on runtime.
Note: Already raised the issue with pycharm but haven't got any solution yet
